operator/src/test/java/org/bf2/operator/operands/AdminServerTest.java

```diff
@@ -82,12 +77,17 @@ static ManagedKafka buildBasicManagedKafka(String name, String strimziVersion, T
 
     @ParameterizedTest
     @CsvSource({
-        "test-mk-q,     1, true,  false, /expected/adminserver.yml",
-        "test-mk-tls-q, 2, true,  true,  /expected/adminserver-tls.yml"
+        "test-mk-q,     1, true,  false, false, /expected/adminserver.yml",
+        "test-mk-tls-q, 2, true,  true, false, /expected/adminserver-tls.yml",
+        "test-mk-q,     2, true,  false, true, /expected/adminserver-affinity.yml"
     })
-    void createAdminServerDeployment(String name, String versionString, boolean quarkusBased, boolean tls, String expectedResource) throws Exception {
+    void createAdminServerDeployment(String name, String versionString, boolean quarkusBased, boolean tls, boolean useNodeAffinity, String expectedResource) throws Exception {
         ManagedKafka mk = buildBasicManagedKafka(name, versionString, tls ? new TlsKeyPair() : null);
 
+        if (useNodeAffinity) {
+            CanaryTest.useNodeAffinity(mk);
+        }
```

Comment:
This seems a little confusing. Might be better to put `useNodeAffinity` in some test util class or similar.

Reply:
When everything is in the same package I'm not too concerned about having separate utility classes.  If you prefer one, it can of course be added.

Reply:
Thanks for adding that, looks good.